MobileSyncExplorerKotlinTemplate/app/build.gradle.kts

```diff
@@ -6,34 +6,34 @@ plugins {
 }
 
 dependencies {
-    implementation("androidx.core:core-ktx:1.7.0")
-    implementation("androidx.compose.ui:ui:1.4.0-alpha02")
-    implementation("androidx.compose.material:material:1.4.0-alpha02")
-    implementation("androidx.compose.ui:ui-tooling-preview:1.4.0-alpha02")
-    implementation("androidx.lifecycle:lifecycle-runtime-ktx:2.4.0")
-    implementation("androidx.activity:activity-compose:1.4.0")
-    implementation("androidx.window:window:1.0.0")
+    implementation("androidx.core:core-ktx:1.9.0")
+    implementation("androidx.compose.ui:ui:1.6.0-alpha08")
+    implementation("androidx.compose.material:material:1.6.0-alpha08")
+    implementation("androidx.compose.ui:ui-tooling-preview:1.6.0-alpha08")
+    implementation("androidx.lifecycle:lifecycle-runtime-ktx:2.6.2")
+    implementation("androidx.activity:activity-compose:1.8.0")
+    implementation("androidx.window:window:1.1.0")
 
-    implementation("com.salesforce.mobilesdk:MobileSync:11.1.0")
+    implementation("com.salesforce.mobilesdk:MobileSync:11.0.1")
```

Comment:
I know in dev it does not matter since we use composite build, but we should have the latest version here still.